test/acceptance/tests/basic/basic_test.go

```diff
@@ -64,6 +64,11 @@ func TestBasic(t *testing.T) {
 	randomSuffix := strings.ToLower(random.UniqueId())
 	tfVars := suite.Config().TFVars("route_table_ids")
 	tfVars["suffix"] = randomSuffix
+	serverServiceName := "custom_test_server"
+	// This uses the explicitly passed service name rather than the task's family name
```

Comment:
nit: to follow style guidelines for go comments.
```suggestion
	// This uses the explicitly passed service name rather than the task's family name.
```